### PR TITLE
Use drmIsMaster() from libdrm when it is provided

### PR DESCRIPTION
**Pull request by RAOF**
Closes: #813

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -282,11 +282,18 @@ pkg_check_modules(XCB_COMPOSITE REQUIRED xcb-composite)
 pkg_check_modules(XCB_XFIXES REQUIRED xcb-xfixes)
 pkg_check_modules(XCB_RENDER REQUIRED xcb-render)
 pkg_check_modules(X11_XCURSOR REQUIRED xcursor)
+pkg_check_modules(DRM REQUIRED libdrm)
```
Comment (AlanGriffiths) [inline CMakeLists.txt]: Will requiring drm cause problems for UBports builds targetting libhybris/android? @mariogrip

Comment (RAOF) [reply CMakeLists.txt]: As I mentioned in the first commit message, we already [unconditionally require libdrm](https://github.com/MirServer/mir/blob/34f92104efea90aaa79f90d9acd5b9f638f3d21f/src/server/console/minimal_console_services.cpp#L32
); we just don't unconditionally check for it.

Comment (AlanGriffiths) [reply CMakeLists.txt]: Ack. Makes sense.

File: src/server/console/minimal_console_services.cpp [hunk 2]
```diff
@@ -37,7 +37,7 @@
 
 // TODO: once libdrm provides this symbol, we need a way to set MIR_LIBDRM_HAS_IS_MASTER
```
Comment (AlanGriffiths) [inline src/server/console/minimal_console_services.cpp]: Comment needs updating to reflect these changes.

**Comment by wmww** [comment]
bors r=AlanGriffiths

**Pull request merged**